tox.ini

```diff
@@ -121,6 +121,22 @@ pip_pre =
     predeps: true
     !predeps: false
 
+# Test that we can build astropy with the latest developer versions of all
+# build-time dependencies,
+[testenv:devbuild]
+skip_install = true
+changedir = .tmp/{envname}
+setenv =
+    PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+deps =
+    numpy>=0.0.dev0
+    git+https://github.com/pypa/setuptools.git
+    git+https://github.com/pypa/setuptools_scm.git
+    git+https://github.com/cython/cython
+    git+https://github.com/astropy/extension-helpers
```

Comment:
Would it help to include `pyerfa` here too as `git+https`?

Reply:
No, it is not a build-time dependency

Reply:
OK, makes sense!